package.json

```diff
@@ -2,26 +2,47 @@
   "name": "idbify",
   "version": "1.0.0",
   "description": "A simple interface for IDB with promises, you can define your schema up front and then it provides put, delete, and get functions.",
-  "main": "dist/idbify.es5.js",
-  "module": "dist/idbify.min.js",
+  "main": "dist/idbify.umd.js",
+  "module": "dist/idbify.module.js",
   "repository": "https://github.com/coltpini/idbify.git",
   "author": "Colt Pini <coltpini@github.com>",
   "license": "Apache 2.0",
+  "babel": {
+    "minified": true,
```

Comment:
What's the reason we're outputting minified result by default? This library is already very small, and people's packaging setups usually minify their bundles as a whole.

Reply:
No reason other than I didn't want to depart too far from what was already here.

Reply:
Dropped the minification.